README.md

```diff
@@ -104,4 +104,83 @@ Header: Content-Type:"application/cloudevents+json; charset=UTF-8"
     },
     "aliyuneventbusname":"demo"
 }
-```
\ No newline at end of file
+```
+* Use HttpSource to put events
+
+EventBridge HttpSource allows you to put events to eventbus in the form of webhook.
+
+Here is an example explaining how to put events using EventBridge HttpSource.
+
+1. Create an EventBridge HttpSource
+
+    - eventSourceName: Name of EventSource
+    - eventBusName: Name of EventBus
+    - description: Description
+    - className: HttpEvent. This parameter is a fixed value and cannot be modified.
+    - config: HttpSource Config
+    - Type: Request type. Available values are 'HTTP', 'HTTPS' and 'HTTP&HTTPS'.
+    - Method: Allowed HTTP request methods. The request will be filtered if the http request method type for accessing the webhook does not meet the configuration.
+    - SecurityConfig: Security configuration type. Available values are 'none', 'ip' and 'referer'.
+    - Ip: IP security configuration. Http requests whose source ip is not in the configured network segment will be filtered if the security configuration is selected as 'ip'.
+    - Referer: Referer security configuration. HTTP requests whose referer is not in this configuration will be filtered if the security configuration is selected as 'referer'.
+
+A webhook will be generated after the creation of HttpSource.
+```
+http://127.0.0.1:7001/source/createEventSource
+```
+```json
+{
+  "eventSourceName": "httpEventSourceDemo",
+  "eventBusName": "demo",
+  "description": "http source demo",
+  "className": "HttpEvent",
+  "config": {
+    "Type": "HTTP&HTTPS",
+    "Method": ["GET", "POST"],
+    "SecurityConfig": "ip",
+    "Ip": ["10.0.0.0/8"],
+    "Referer":[]
+  }
+}
+```
+2. Put event to EventBus
+
+Http request to access this webhook will be converted into a CloudEvent and delivered to eventbus.
+
+```
+http://127.0.0.1/webhook/putEvents?token=43146d108b2123
+Header: Content-Type:"application/json"
+```
+```json
+{
```

Comment:
Provide a DEMO of HTTP CURL? which match the cloudevent result.

Reply:
done